spec/controllers/transactions_controller_spec.rb

```diff
@@ -7,6 +7,8 @@
   let(:user) { FactoryGirl.create(:user) }
 
   describe "GET #in_review" do
+    before(:all) { Settings.billing.review_period = 7.days }
```

Comment:
We should probably make sure to reset this after the test so we don't run into spec order issues.